test/controllers/result_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class ResultControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get result_index_url
```

Comment:
I don't think you have this controller anymore, lets remove this test.

Reply:
Removed.